plugins/api-docs/package.json

```diff
@@ -44,8 +44,9 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "4.0.0-alpha.57",
     "graphiql": "^1.8.8",
-    "graphql-ws": "^5.4.1",
     "graphql": "^16.0.0",
+    "graphql-ws": "^5.4.1",
+    "grpc-docs": "^1.1.2",
```

Comment:
Not hyped about the look of this dep tbh, don't really want to add it to everyones trees at this point. What do you think about perhaps making this a module instead that provides an easy way to add this rendering method to the widget API?

Reply:
Makes sense, and is reasonable ask. I'll move it to a module. What do you mean by an easy way of adding this rendering method to the widget API ? Wouldn't it be a added like this: https://github.com/backstage/backstage/tree/master/plugins/api-docs#custom-api-renderings ?

Reply:
Was gonna say you could provide a factory too, but realized that would create a dependency on the plugin package, so perhaps not. Would hope that we could simply export a component from the separate module that matches `ApiDefinitionWidget` and that the module doesn't need a dependency on the plugin.

Proper way would otherwise be a `*-react` package, but ain't nobody got time for that